frontend/src/api.ts

```diff
@@ -80,3 +80,15 @@ interface MeResponse {
 export function me(): Promise<MeResponse> {
   return apiCall<MeResponse>('/api/me');
 }
+
+export type Org = {
```

Comment:
let be consisted with interface/type. In most cases they are interchangeable so I don't really have any preferences here. But if you prefer type instead of an interface please update all responses in this file.

Reply:
+1 for consistency. Done in 8de2924, please check this is the right syntax for an array interface,